### DISCUSSION TITLE
How to stop Georgia-Reborn (or perhaps Foobar2000?) from using images inside the album folders as album covers?

### DISCUSSION THREAD

**Discussion by azagthoth1** [TT-ReBORN/Georgia-ReBORN#154] [Q&A] [answered]
This used to not be a problem, but I updated both Georgia-ReBORN and Foobar2000, and now I noticed that in some albums in my library, which I use configured as Views > Albums, the image showing as the album cover when I'm on the "Details" tab, or when I'm on the Library tab, but, like, not the first one, when you click on it and it  shows all of your albums, but, like, when you're playing an album, and click on "Library" again, and it switches to "inside" the album, with its tracklist showing on the right side (Sorry, english is not my first language. Is it called "playlist view"? I don't know), you know? When I'm on THAT page, it does not show the intended album cover either, and it used to, because prior to moving newly downloaded albums into my music directory, I always edit the tags with Mp3tag and put the album cover there, so I don't even need having files on the folder for the album covers to be shown, but sometimes I do like keeping "scans" folders inside some albums, with pics of the booklet, back cover, etc, BUT, as I said in the beginning, this used to not be a problem, because these subfolders wouldn't get detected, but now, it seems that they are. 

Images speak louder than words, so let me show you the issue:
Here you see the album on the Library tab with its correct cover.

![Print 1](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/35978048/1b40acba-465f-4cc4-ad0c-b6ba1da7a144)

But when I click on Library again (or Details), this is what I see, which is the booklet-1.jpeg file, inside the "scans" folder inside the album folder: 

![Print 2](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/35978048/bfa3b905-ed2e-4cdc-abab-8c08b1fe82b1)

When I remove the "scans" folder, it comes back to the intended album cover, but I don't know which change prompted this to start, I just know that this used to not be a problem, and there's so many options, I just don't know where I could possibly find the fix. Excuse me if it's more of a Foobar2000-related issue than your theme, per se. Perhaps you could help me anyway.

**Comment by TT-ReBORN** [comment, marked as answer]
Hi @azagthoth1,

the correct structure to display album covers is the following:

**In your music album root folder:**

**Covers:**
cover.jpg/png, cover2.jpg/png, cover3.jpg/png etc...
folder.jpg/png, folder2.jpg/png, folder3.jpg/png etc...
front.jpg/png, front2.jpg/png, front3.jpg/png etc...

**cdART:**
cd.png, cd2.png, cd3.png etc...

**vinylART:**
vinyl.png, vinyl2.png, vinyl3.png etc...

**Booklet ( can have subfolders named "Artwork" or "Images" or "Scans" ):**
booklet.jpg/png, booklet2.jpg/png, booklet3.jpg/png etc...
artwork.jpg/png, artwork2.jpg/png, artwork3.jpg/png etc...
image.jpg/png, image2.jpg/png, image3.jpg/png etc...
scan.jpg/png, scan2.jpg/png, scan3.jpg/png etc...
back.jpg/png, back2.jpg/png, back3.jpg/png etc...

**Multi CD's ( can have subfolders named "CD 1", "CD 2", "CD 3" etc ).**

---

If you do not have `cover.jpg/png` or `folder.jpg/png` or `front.jpg/png` it will start using the images from the booklet directories.
It seems you are embedding your album covers directly in your music files and need to use the embedded album art option for
Georgia-ReBORN and for foobar2000.

**To display embedded album art, activate:** 
In Georgia-ReBORN via top menu `Options` > `Player controls` > `Album art` > `Load embedded album art first`.
In foobar's `Preferences` > `Advanced` > `Display` > `Album art` > `Embedded vs external` > `Prefer embedded`.
Restart foobar and it should now display the album art correctly.

~~**Note:** When displaying embedded album art, you won't be able to display booklet images when you mouse scroll 
on the big album cover.~~

-TT

**Comment by azagthoth1** [comment]
> **To display embedded album art, activate:**
> In Georgia-ReBORN via top menu `Options` > `Player controls` > `Album art` > `Load embedded album art first`.
> In foobar's `Preferences` > `Advanced` > `Display` > `Album art` > `Embedded vs external` > `Prefer embedded`.
> Restart foobar and it should now display the album art correctly.

That was it! Thank you very much!!! <3 <3 <3 

> Note: When displaying embedded album art, you won't be able to display booklet images when you mouse scroll
on the big album cover.

I didn't even know about that! I only use it on the normal size though.

**Reply by TT-ReBORN** [reply]
Just checked myself and corrected my answer. Actually I changed quite a few times the image loading method, so excuse me.
You still will be able to mouse scroll on the big album art through images in your music folder when embedded album art is active.
